build/release.js

```diff
@@ -272,11 +280,28 @@ function writeJsonSync( fname, json ) {
 	}
 }
 
-function copy( oldFile, newFile ) {
-	status( "Copying " + oldFile + " to " + newFile );
-	if ( !dryrun ) {
-		fs.writeFileSync( newFile, fs.readFileSync( oldFile, "utf8" ) );
+function fixMinRef( oldText ) {
+	var mapRef = new RegExp( "^//# sourceMappingURL=jquery-migrate.min.map\\n?", "m" );
+
+	// Remove the ref for now rather than try to fix it
+	var newText = oldText.replace( mapRef, "" );
+	if ( oldText === newText ) {
+		throw "fixMinRef: Unable to find the sourceMappingURL";
```

Comment:
Let's throw errors, not strings so that stack traces are preserved:
```suggestion
		throw Error( "fixMinRef: Unable to find the sourceMappingURL" );
```